pkgs/applications/science/biology/bppsuite/default.nix

```diff
@@ -0,0 +1,18 @@
+{ stdenv, fetchFromGitHub, cmake, bpp-core, bpp-seq, bpp-phyl, bpp-popgen }:
+
+stdenv.mkDerivation rec {
+  pname    = "bppsuite";
+
+  inherit (bpp-core) meta version;
+
+  src = fetchFromGitHub {
+    owner  = "BioPP";
+    repo   = pname;
+    rev    = "v${version}";
+    sha256 = "1wdwcgczqbc3m116vakvi0129wm3acln3cfc7ivqnalwvi6lrpds";
+  };
```

Comment:
Let's avoid this style of spacing. Among other things, it can cause unnecessarily larger diffs and is harder for robots or sed scripts to maintain.  You can run this through [nixpkgs-format](https://github.com/nix-community/nixpkgs-fmt) to fix.

Reply:
Sure, fixed.